strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/artifact/coordinates/PypiWheelArtifactCoordinates.java

```diff
@@ -0,0 +1,239 @@
+package org.carlspring.strongbox.artifact.coordinates;
+
+import javax.persistence.Entity;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.Map;
+
+import org.carlspring.strongbox.util.PypiWheelArtifactCoordinatesUtils;
+import org.semver.Version;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Represents {@link ArtifactCoordinates} for PyPi repository
+ * 
+ * Proper path for this coordinates is in the format of: 
+ * {distribution}-{version}(-{build tag})?-{python tag}-{abi tag}-{platform tag}.whl.
+ * Example: distribution-1.0-1-py27-none-any.whl
+ * 
+ * @author alecg956
+ */
+@Entity
+@SuppressWarnings("serial")
+@XmlRootElement(name = "PypiWheelArtifactCoordinates")
+@XmlAccessorType(XmlAccessType.NONE)
+@ArtifactCoordinatesLayout(name = PypiWheelArtifactCoordinates.LAYOUT_NAME, alias = PypiWheelArtifactCoordinates.LAYOUT_ALIAS)
+public class PypiWheelArtifactCoordinates
+    extends AbstractArtifactCoordinates<PypiWheelArtifactCoordinates, Version>
+{
+
+    public static final String LAYOUT_NAME = "PyPi";
+    public static final String LAYOUT_ALIAS = "pypi";
+
+    public static final String DISTRIBUTION = "distribution";
+    public static final String VERSION = "version";
+    public static final String BUILD_TAG = "build_tag";
+    public static final String LANG_IMPL_VERSION_TAG = "languageImplementationVersion_tag";
+    private static final String ABI_TAG = "abi_tag";
+    private static final String PLATFORM_TAG = "platform_tag";
```

Comment:
I'm not quite sure why part of the coordinates are `private static final` whereas the others are `public static final`... I believe they should all be the latter.

Reply:
I'm not sure either, will change them to all be public.